jams/schemata/namespaces/pitch/note_midi.json

```diff
@@ -0,0 +1,8 @@
+{"note_midi":
+    {
+        "value": { "type": "number" },
```

Comment:
So that immediately breaks the 0-127 range, which makes me think we should leave the spec as is, and not worry about bounds at all.  (Logarithms are unbounded anyway, yolo.)